composer.json

```diff
@@ -37,11 +39,5 @@
         "psr-4": {
             "WyriHaximus\\CakePHP\\Tests\\MinifyHtml\\View\\Helper\\": "tests/"
         }
-    },
-    "config": {
-        "sort-packages": true,
-        "platform": {
-            "php": "5.6"
-        }
```

Comment:
Can you revert this, this is in to ensure everything we require works with the lowest supported PHP version.